server-common/src/main/java/org/apache/kafka/server/common/KRaftVersion.java

```diff
@@ -28,6 +28,8 @@ public enum KRaftVersion implements FeatureVersion {
 
     public static final String FEATURE_NAME = "kraft.version";
 
+    public static final KRaftVersion LATEST_PRODUCTION = KRAFT_VERSION_1;
```

Comment:
Currently, the latest production of KRaftVersion is 0. So, is this change expected?

Reply:
This is also not correct. 
https://github.com/apache/kafka/blob/6a43a6bddfa04a15832ccccf8742919d1666fd86/server-common/src/main/java/org/apache/kafka/server/common/KRaftVersion.java#L27